packages/zowe-explorer-api/src/vscode/doc/VscSettings.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+import * as vscode from "vscode";
+import * as imperative from "@zowe/imperative";
+
+export class VscSettings {
+    /**
+     * Retrieves a generic setting either in user or workspace.
+     * @param {string} key - The config property that needs retrieving
+     * @param {T} defaultValue - Default value if config property is undefined
+     */
+    public static getDirectValue<T>(key: string, defaultValue?: T): T {
+        const [first, ...rest] = key.split(".");
+        return vscode.workspace.getConfiguration(first).get(rest.join("."), defaultValue);
+    }
```

Comment:
This is already accessible in `SettingsConfig` - I'm wondering if we should move these two functions into that class?

Reply:
Hey @traeok SettingsConfig is in the Zowe Explorer package, this new is in the ZE API package.  I was thinking to create a follow up story to have ZE use the ZE API instance to remove duplication.

Reply:
https://github.com/zowe/zowe-explorer-vscode/issues/3304

Reply:
Thanks Billie!